base/random.jl

```diff
@@ -1596,6 +1618,30 @@ end
 # each element of A is included in S with independent probability p.
 # (Note that this is different from the problem of finding a random
 #  size-m subset of A where m is fixed!)
+
+"""
+    randsubseq!([rng::AbstractRNG=GLOBAL_RNG,] S, A, p)
```

Comment:
Somewhere else, ` -> S` is used at the end of the signature to indicate which object is returned.